test_existing_projects.sh

```diff
@@ -15,3 +15,4 @@
 #  limitations under the License.
 
 go test -v ./test/projects/memcached-api-server
```

Comment:
we can add a for loop over directory listing under `./test/projects` so that future addition of test-projects don't require change in this file.

Though Not a big deal because we are not adding test projects that frequently :)